python/tests/catalog/test_glue.py

```diff
@@ -67,7 +67,18 @@ def get_random_databases(n: int) -> Set[str]:
 
 @pytest.fixture(name="_bucket_initialize")
 def fixture_s3_bucket(_s3) -> None:  # type: ignore
-    _s3.create_bucket(Bucket=BUCKET_NAME)
+    bucket = _s3.create_bucket(Bucket=BUCKET_NAME)
+    yield bucket
```

Comment:
Does this change the return type?

Reply:
No, this is a bit of an interesting fixture. It didn't return anything before, and now it just blocks the yield until the test is passed, and then cleans it up.